include/CL/opencl.hpp

```diff
@@ -8859,6 +8931,54 @@ typedef CL_API_ENTRY cl_int (CL_API_CALL *PFN_clEnqueueReleaseD3D10ObjectsKHR)(
     {
         return detail::errHandler(::clFinish(object_), __FINISH_ERR);
     }
+
+#if CL_HPP_TARGET_OPENCL_VERSION >= 300
+    cl_int enqueueWaitSemaphoresKHR(
+        const vector<SemaphoreKhr>* sema_objects,
+        const vector<cl_semaphore_payload_khr>* sema_payloads = NULL,
```

Comment:
Could we do better than a pointer-to-vector here too?  Maybe:

```suggestion
        const vector<cl_semaphore_payload_khr>& sema_payloads = {},
```

Reply:
Done, thank you!